resources/assets/js/authorizations.js

```diff
@@ -2,7 +2,7 @@ let user = window.App.user;
 
 module.exports = {
     owns (model, prop = 'user_id') {
-        return model[prop] === user.id;
+        return parseInt(model[prop]) === user.id;
```

Comment:
It's best to specify the `radix`, like [suggested on MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt) 🙂

```js
return parseInt(model[prop], 10) === user.id;
```